src/battle_script_commands.c

```diff
@@ -14998,8 +14998,16 @@ static void Cmd_handleballthrow(void)
             }
             else
             {
-                odds = Sqrt(Sqrt(16711680 / odds));
-                odds = 1048560 / odds;
+                if (P_CATCH_CURVE >= GEN_6)
+                {
+                    odds = 16581375 / (odds * odds * odds);
```

Comment:
16581375 seems to be `255*255*255`. I think it's more obvious what's going on (`255/odds` cubed) if you write it like that.

Reply:
Yeah but rounding errors. Hence why I did it this way

Reply:
I could make a comment but the compiler will generate different code and that code has more rounding errors than this one

Reply:
How could the compiler generate different code? Those two things are identical semantically. Do you have a Godbolt link showing a difference between the compiled code, and an example `odds` value which has different rounding behavior?

Reply:
I am really glad you asked!



> How could the compiler generate different code? Those two things are identical semantically. Do you have a Godbolt link showing a difference between the compiled code, and/or an example `odds` value which has different rounding behavior?
> 
> EDIT: [Godbolt showing that they compile to the same thing](https://godbolt.org/z/x7Y8sb1Te).

I misread your comment. I thought you were suggesting 

odds = 255 / odds;

odds = odds * odds *odds;